client/bin/multi-client.js

```diff
@@ -385,9 +385,12 @@ class NonInteractiveState {
 						deviceUrl = device;
 					}
 					let status = await rp.get(
-						new url.URL('/state', deviceUrl).toString(),
+						new url.URL('/start', deviceUrl).toString(),
```

Comment:
Should we not keep the state endpoint in case something else was using it?

What if we want to use the state endpoint for update locks or something that doesn't use the worker.

Reply:
state endpoint still exists on the worker, currently on the worker side all it does is return whether its BUSY or IDLE. This PR doesn't touch what endpoints are available, only starts using /start instead of /state then /start

In the worker logic, if something tries the /start endpoint and something is already using it, it returns BUSY